src/components/modals/RoleModal.vue

```diff
@@ -11,15 +8,41 @@
           : "bluffing"
       }}
     </h3>
-    <ul class="tokens">
-      <li
-        v-for="role in availableRoles"
-        :class="[role.team]"
-        :key="role.id"
-        @click="setRole(role)"
-      >
-        <Token :role="role" />
+    <ul>
+      <li class="button-group" v-if="isBluffs()">
```

Comment:
I would move this to the bottom of the modal. In almost all cases the user doesn't need to access the extra travelers. On top of that, it needs to be hidden for `session.isSpectator = true` (aka "player in a live session"), because they can't assign travelers.